CONTRIBUTING.rst

```diff
@@ -58,211 +56,173 @@ Get Started!
+
+    .. code-block:: console
 
-        $ conda install -n base conda-libmamba-solver
-        $ conda config --set solver libmamba
+        conda install -n base conda-libmamba-solver
+        conda config --set solver libmamba
 
     For more information, please see the following link: https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community
 
     Alternatively, you can use the `mamba <https://mamba.readthedocs.io/en/latest/index.html>`_ package manager, which is a drop-in replacement for ``conda``. If you are already using `mamba`, replace the following commands with ``mamba`` instead of ``conda``.
 
 Ready to contribute? Here's how to set up ``xdatasets`` for local development.
 
-#. Fork the ``xdatasets`` repo on GitHub.
-#. Clone your fork locally::
-
-    $ git clone git@github.com:your_name_here/xdatasets.git
-
-#. Install your local copy into a development environment. You can create a new Anaconda development environment with::
-
-    $ conda env create -f environment-dev.yml
-    $ conda activate xdatasets
-    $ flit install --symlink
-
-  This installs ``xdatasets`` in an "editable" state, meaning that changes to the code are immediately seen by the environment.
-
-#. To ensure a consistent coding style, install the ``pre-commit`` hooks to your local clone::
-
-    $ pre-commit install
-
-  On commit, ``pre-commit`` will check that ``black``, ``blackdoc``, ``isort``, ``flake8``, and ``ruff`` checks are passing, perform automatic fixes if possible, and warn of violations that require intervention. If your commit fails the checks initially, simply fix the errors, re-add the files, and re-commit.
+#. First, clone the ``xdatasets`` repo locally.
 
-  You can also run the hooks manually with::
+    * If you are not a ``xdatasets`` collaborator, first fork the ``xdatasets`` repo on GitHub, then clone your fork locally.
 
-    $ pre-commit run -a
+        .. code-block:: console
 
-  If you want to skip the ``pre-commit`` hooks temporarily, you can pass the ``--no-verify`` flag to `$ git commit`.
+            git clone git@github.com:your_name_here/xdatasets.git
 
-#. Create a branch for local development::
+    * If you are a ``xdatasets`` collaborator, clone the ``xdatasets`` repo directly.
 
-    $ git checkout -b name-of-your-bugfix-or-feature
+        .. code-block:: console
 
-  Now you can make your changes locally.
+            git clone git@github.com:``sebastienlanglois``/xdatasets.git
 
-#. When you're done making changes, we **strongly** suggest running the tests in your environment or with the help of ``tox``::
+#. Install your local copy into a development environment. You can create a new Anaconda development environment with:
 
-    $ python -m pytest
-    # Or, to run multiple build tests
-    $ tox
+    .. code-block:: console
 
-#. Commit your changes and push your branch to GitHub::
+        conda env create -f environment-dev.yml
+        conda activate xdatasets-dev
```

Comment:
```suggestion
        conda activate xdatasets
```

Reply:
The environment here is named `xdatasets` in the `environment-dev` file.

Reply:
Right. That's not the case everywhere. No need to push that change up.